docs/Software Architecture Document/Components View/index.md

```diff
@@ -136,6 +136,19 @@ The view is represented by module and subsystem diagrams that show the system's
   - *Constraints* 
     - Configurable count of threads usage.
 
+#### App Launch
+  - *Responsibility*
+    - Launch known applications on devices.
+  - *Relations*
+    - Composed by ***Application Manager*** 
+    - Use ***Resume Controller*** interface to get HMI level of saved application.
+  - *Interfaces*
+    - Provides ***App Launch Controller*** interface 
+  - *Behavior*
+    - ***App Launch*** launch all known applications on newly connected device.
+  - *Constraints*
+    - N/A
```

Comment:
@AnastasiyaBritanova, sorry, but what exactly should have version higher then 4?